node/node.go

```diff
@@ -1980,6 +1987,19 @@ func (app *App) setupDBs(ctx context.Context, lg log.Log) error {
 		return fmt.Errorf("open sqlite db: %w", err)
 	}
 	app.db = sqlDB
+
+	apiDBLog := app.addLogger(ApiStateDBLogger, lg).Zap()
+	apiSqlDB, err := statesql.Open("file:"+filepath.Join(dbPath, dbFile), []sql.Opt{
+		sql.WithReadOnly(),
+		sql.WithLogger(apiDBLog),
+		sql.WithConnections(app.Config.API.DatabaseConnections),
+		sql.WithMigrationsDisabled(),
+	}...)
```

Comment:
NIT: the slice definition isn't needed 🙂 

```suggestion
	apiSqlDB, err := statesql.Open("file:"+filepath.Join(dbPath, dbFile),
		sql.WithReadOnly(),
		sql.WithLogger(apiDBLog),
		sql.WithConnections(app.Config.API.DatabaseConnections),
		sql.WithMigrationsDisabled(),
	)
```